vision/detect/detect.go

```diff
@@ -14,22 +14,24 @@ import (
 	"io"
 	"os"
 
-	"cloud.google.com/go/vision"
+	"cloud.google.com/go/vision/apiv1"
```

Comment:
Can we use an import rename of "vision" so people don't get confused when they see the `vision.` below?